build/pmm-server-docker/playbook-install.yml

```diff
@@ -280,37 +280,21 @@
       stat: path=/srv/grafana/grafana.db
       register: grafana_db
 
-    - name: Grafana                    | Disable Anonymous access
-      when: not grafana_db.stat.exists
-      ini_file:
-        dest: /etc/grafana/grafana.ini
-        section: auth.basic
-        option: enabled
-        value: true
-
     - name: Grafana                    | Disable Anonymous access
       when: not grafana_db.stat.exists
       ini_file:
         dest: /etc/grafana/grafana.ini
         section: auth.anonymous
         option: enabled
-        value: false
-
-    - name: Grafana                    | Disable Anonymous access
-      when: not grafana_db.stat.exists
-      ini_file:
-        dest: /etc/grafana/grafana.ini
-        section: auth.anonymous
-        option: org_role
-        value: Editor
+        value: 'false'
```

Comment:
Why do you quote it?

Reply:
```
[WARNING]: The value False (type bool) in a string field was converted to
u'False' (type string). If this does not look like what you expect, quote the
entire value to ensure it does not change.
```

Reply:
We also use 'false' and 'true' in Grafana config, not 'False' and 'True'.